src/dap/api.d.ts

```diff
@@ -22,10 +21,7 @@ export namespace Dap {
      * This can either be a normal result ('success' attribute true) or an error response ('success' attribute false and the 'message' set to 'cancelled').
      * Returning partial results from a cancelled request is possible but please note that a frontend client has no generic way for detecting that a response is partial or not.
      */
-    on(
-      request: 'cancel',
-      handler: (params: CancelParams) => Promise<CancelResult | Error>,
-    ): () => void;
+    on(request: 'cancel', handler: (params: CancelParams) => Promise<CancelResult | Error>): () => void;
```

Comment:
Yeah, I think it's because I re-ran the "generate DAP API" task after I updated the custom-dap.json file (adding the toggle requests). I'll re-run gulp format and see if that fixes it

Reply:
I think it's formatted correctly now